src/subscription_manager/repofile.py

```diff
@@ -573,6 +572,11 @@ def __init__(self, path=None, name=None):
         super(ZypperRepoFile, self).__init__(path, name)
         self.gpgcheck = False
         self.autorefresh = False
+        # According to
+        # https://github.com/openSUSE/libzypp/blob/67f55b474d67f77c1868955da8542a7acfa70a9f/zypp/media/MediaManager.h#L394
+        #   the following values are valid: "yes", "no", "host", "peer"
+        self.gpgkey_ssl_verify = 'yes'
+        self.repo_ssl_verify = 'host'
```

Comment:
I would use here default value `None` and when something is read from configuration file, then I would use value from configuration file.